honeybee/logutil.py

```diff
@@ -1,29 +1,42 @@
 import logging
 from logging.handlers import TimedRotatingFileHandler
+import os
 
 
-def get_logger(name):
+def _get_log_folder():
+    home_folder = os.getenv('HOME') or os.path.expanduser('~')
+    log_folder = os.path.join(home_folder, '.honeybee')
+    if not os.path.isdir(log_folder):
+        os.mkdir(log_folder)
+    return log_folder
+
+
+def get_logger(name, filename='honeybee.log'):
     """Get a logger to be used for each module.
 
-    Use this only once in __init__.py file.
+    Args:
+        name: Logger name. The good practice is to set it to __init__ from inside each
+            modules.
+        filename: Logger filename (Default: honeybee.log).
     """
     logger = logging.getLogger(name)
     logger.setLevel(logging.DEBUG)
```

Comment:
Should we not set this to `INFO` and give the option to change the log level to debug when someone is actually trying to debug a script?

Reply:
We can use INFO but again as I commented down there I'm setting this us based on what a user would do more than a developer.

Also this one doesn't really do anything right now as I'm setting the level for each handler separately. File handler is set to DEBUG and stream handler is set to WARNING.  Are you suggesting to set the File handler to INFO?

Reply:
That makes sense. In that case can we set it to debug by default but give the option of setting it to something else? I'm thinking of the pollination stuff which most likely won't need the DEBUG level by default.

Reply:
Sure. I will add two input arguments so one can change the log level for each handler when creating the logger.